bundles/org.openhab.binding.nzwateralerts/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="nzwateralerts"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="wateralert">
+		<label>Alert</label>
+		<description>Water Alert Levels for New Zealand water supply.</description>
+
+		<channels>
+			<channel id="alertlevel" typeId="alertlevel"/>
+		</channels>
+
+		<config-description>
+			<parameter name="location" type="text" required="true">
+				<label>Location</label>
+				<description>The location to get the Water Alert level for.</description>
+				<required>true</required>
```

Comment:
This tag can be removed as it is redundant.

Reply:
Sorry can you clarify which tag to remove?

Reply:
The `required` tag, as it is already present as attribute. The `required` tag is also deprecated.

Reply:
Doh!